### PR TITLE
fix: [ANDROAPP-5413] Review all ignored tests in TeiDashboardTest file

### PR DESCRIPTION
**Pull request by DavidAparicioAlbaAsenjo**
## Description
Please include a summary of the change and include the related jira issue if it exists.

[ jira issue ](https://dhis2.atlassian.net/browse/ANDROAPP-

## Solution description
If this PR is a fix include a brief description on how the issue is solved.
## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [ ] Smartphone Emulator
- [ ] Tablet Emulator
- [ ] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [ ] 8.X
- [ ] 9.X - 10.X
- [ ] 11.X - 13.X
- [ ] Other
## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: app/src/androidTest/java/org/dhis2/usescases/teidashboard/TeiDashboardTest.kt [hunk 1]
```diff
@@ -33,7 +33,7 @@ class TeiDashboardTest : BaseTest() {
 
     @get:Rule
     val composeTestRule = createComposeRule()
-    @Ignore("SDK related, also indeterminate")
+    //@Ignore("SDK related, also indeterminate")
```
Comment (xavimolloy) [inline app/src/androidTest/java/org/dhis2/usescases/teidashboard/TeiDashboardTest.kt:36]: Must remove all commented code

File: app/src/main/java/org/dhis2/App.java [hunk 2]
```diff
@@ -117,7 +116,9 @@ public void onCreate() {
         MapController.Companion.init(this);
 
         setUpAppComponent();
-        Timber.plant(BuildConfig.DEBUG ? new DebugTree() : new ReleaseTree(appComponent.injectCrashReportController()));
+        if(BuildConfig.DEBUG){
```
Comment (xavimolloy) [inline app/src/main/java/org/dhis2/App.java:119]: There are changes in the PR that are not related to the issue, remember rebase with develop

**Review by xavimolloy — CHANGES_REQUESTED** [review]
PR was on Master, remove commented code and rebase branch

File: app/src/androidTest/java/org/dhis2/usescases/teidashboard/TeiDashboardTest.kt [hunk 3]
```diff
@@ -151,12 +149,11 @@ class TeiDashboardTest : BaseTest() {
             clickOnFirstReferralEvent()
             clickOnReferralOption()
             clickOnReferralNextButton()
-            checkEventCreatedToastIsShown()
+            // checkEventCreatedToastIsShown()
```
Comment (andresmr) [inline app/src/androidTest/java/org/dhis2/usescases/teidashboard/TeiDashboardTest.kt:152]: please, remove commented code

File: app/src/androidTest/java/org/dhis2/usescases/teidashboard/TeiDashboardTest.kt [hunk 4]
```diff
@@ -168,7 +165,7 @@ class TeiDashboardTest : BaseTest() {
             clickOnScheduleNew()
             clickOnFirstReferralEvent()
             clickOnReferralNextButton()
-            checkEventCreatedToastIsShown()
+          //  checkEventCreatedToastIsShown()
```
Comment (andresmr) [inline app/src/androidTest/java/org/dhis2/usescases/teidashboard/TeiDashboardTest.kt:168]: Please, remove commented code

**Review by andresmr — CHANGES_REQUESTED** [review]

**Review by andresmr — APPROVED** [review]

**Review by xavimolloy — APPROVED** [review]

**Pull request merged**